src/index.html

```diff
@@ -11,7 +11,152 @@
+            name="Name"
+            minlength="1"
+            maxlength="100"
+            required
+          >
+        </label>
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="Age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="Birth date"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="Accept term"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="field">
+          Password:
+          <input
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cat"
+            value="Yes"
+          > Yes
+          <input
+            type="radio"
+            name="cat"
+            value="no"
+          > No
+        </label>
+        <label class="field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="go-to-bed-time"
+          >
+        </label>
+        <label class="field">
+          What are your favorite brand of cars?
+          <select
+          name="cars"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="field">
+          How do you rate  our work?
+          <input
+            type="range"
+            name="work-rate"
+            min="1"
+            max="100"
+            value="1"
+            step="1"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+        <label class="field">
+          Comments:
+          <textarea
+            name="Additional-comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
+        </label>
+        <label class="field">
+          Would you recommend us?
+          <select
+          name="reccomendation"
+          >
```

Comment:
if you have one or two attributes you can write them in one line 
```suggestion
          <select name="reccomendation">
```